test/array_contraints_test.rb

```diff
@@ -9,4 +9,6 @@
   rejects 123
   rejects [123, 456]
   rejects ['abc', 123]
+  rejects nil
```

Comment:
something like that, before you do more work I'll get some feedback from @staugaard maybe he had already planed for that somehow in the original design